src/foam/nanos/fs/fileDropZone/FileDropZone.js

```diff
@@ -277,7 +297,7 @@ foam.CLASS({
           });
         }
       }
-      this.selected = this.files.length - 1;
+      if ( this.selected ) this.selected = this.files.length - 1;
```

Comment:
Dose selected has default value? Can it be undefined?

Reply:
It does not. It is undefined by default and remains undefined unless we pass it a slot in the viewSpec. I added this check because there are places where the dropzone is used without the preview and it doesn't make sense for a card to be selected in that case